.github/workflows/CI.yml

```diff
@@ -717,6 +717,7 @@ jobs:
         run: |
           echo '[patch.crates-io]' >> bindings/Cargo.toml
           ./scripts/cargo/patch-section.sh >> bindings/Cargo.toml
+          cd bindings && cargo update
```

Comment:
This line was needed to apply patch dependencies correctly.

Rust warned:
```
If the patch has a different version from what is locked in the Cargo.lock file, 
run `cargo update` to use the new version.
```

In fact, cargo used the version in Cargo.lock file when building for node-binding test.